src/BaselineOfNewTools/BaselineOfNewTools.class.st

```diff
@@ -164,6 +166,9 @@ BaselineOfNewTools >> baseline: spec [
 			group: #SettingsBrowser with: #(
 						'NewTools-SettingsBrowser'
 						'NewTools-SettingsBrowser-Tests');
+			"Compression Utilities"
+			group: #CompressionUtilities with: #(
```

Comment:
```suggestion
			group: 'CompressionUtilities' with: #(
```

Reply:
I see there are several groups that are using symbols now (it was  not like that before). Since you are there, can you align them too?

Reply:
I did it from the github PR. You're here too :)